user/pages/03.commerce2/02.developer-guide/09.promotions/02.create-a-condition/docs.md

```diff
@@ -4,4 +4,7 @@ taxonomy:
     category: docs
 ---
 
+See [Core Components / Conditions](https://docs.drupalcommerce.org/commerce2/developer-guide/core/conditions)
```

Comment:
You should be able to link to pages internally rather than by the whole domain and everything.